### DISCUSSION TITLE
Does this Support the Trustmaster TX?

### DISCUSSION THREAD

**Discussion by SuperDog34** [berarma/oversteer#81] [Q&A]
Does this Support the Trustmaster TX?

**Comment by berarma** [comment]
I could add it to the list of recognized devices but it would need a device driver to work. I don't know of any device driver for that wheel. It could work though except for the FFB.

**Reply by leillo1975** [reply]
Don't work with tmdrv?
https://github.com/her001/tmdrv

**Reply by berarma** [reply]
> Don't work with tmdrv? https://github.com/her001/tmdrv

Maybe but that project is old, it might be no longer needed with newer kernels, and it doesn't provide FFB.
